scipy/stats/_qmc.py

```diff
@@ -1377,21 +1390,41 @@ def random(self, n: IntNumber = 1) -> np.ndarray:
         """
         sample = np.empty((n, self.d), dtype=float)
 
+        if n == 0:
+            return sample
```

Comment:
Actually it is needed because the `_draw` method would be called with `n-1=-1` which would overflow due to the change from `int` to `uint` (`OverflowError: can't convert negative value to npy_uint64`). It was working before but was accidental. Going into the function and doing nothing.